CRM/Contact/Form/Search/Custom/Proximity.php

```diff
@@ -64,13 +64,13 @@ public function __construct(&$formValues) {
       }
 
       // use the address to get the latitude and longitude
-      CRM_Utils_Geocode_Google::format($this->_formValues);
+      CRM_Core_BAO_Address::addGeocoderData($this->_formValues);
 
       if (!is_numeric(CRM_Utils_Array::value('geo_code_1', $this->_formValues)) ||
         !is_numeric(CRM_Utils_Array::value('geo_code_2', $this->_formValues)) ||
         !isset($this->_formValues['distance'])
       ) {
-        CRM_Core_Error::fatal(ts('Could not geocode input'));
+        throw new CRM_Core_Exception(ts('Could not geocode input'));
```

Comment:
@eileenmcnaughton should we be translating Exception text? ping @mlutfy

Reply:
No, we shouldn't. I grepped to spot-check, and none of the several examples I reviewed translated the text.